libs/templates/src/lib/home/sections/cta/index.tsx

```diff
@@ -0,0 +1,12 @@
+import { FluidContainer } from '@deriv/quill-design';
+import { CTABlockContent } from './data';
+
+const CTASection = () => {
+  return (
+    <FluidContainer className="my-2400 flex-none lg:my-3600">
```

Comment:
these classnames should be semantics, why are we using core values here? 
if there is a reason for this, we should have a call and talk about it.

Reply:
i'll update @mohsen-deriv